api/ciprovider.go

```diff
@@ -0,0 +1,50 @@
+package api
+
+import (
+	"fmt"
+
+	"github.com/airware/vili/circleci"
+	"github.com/airware/vili/config"
+	"github.com/airware/vili/environments"
+	"github.com/airware/vili/kube"
+	"github.com/airware/vili/log"
+	"github.com/airware/vili/slack"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// InitializeCiClient initializes the CI client
+func InitializeCiClient(ciProvider string) error {
+	switch ciProvider {
+	case "circleci":
+		circleci.Init(&circleci.Config{
+			Token:   config.GetString(config.CircleciToken),
+			BaseURL: config.GetString(config.CircleciBaseurl),
+		})
+	default:
+		log.WithField("CiProvider", ciProvider).Warn("You can configure your Ci webhook here with param: CiProvider in config file")
```

Comment:
This can probably return a new error, such as fmt.Errorf("Unsupported ci provider: %s", ciProvider)

Reply:
I think we aren't returning any errors for unsupported CIs. @brianwallace

Reply:
👍